functions.php

```diff
@@ -27,4 +27,21 @@ function ppm_log($expression, $_ = null)
     ob_start();
     call_user_func_array('var_dump', func_get_args());
     file_put_contents('php://stderr', ob_get_clean() . PHP_EOL, FILE_APPEND);
+}
+
+function pcntl_enabled()
+{
+    if (!function_exists('pcntl_signal')) {
+        return false;
+    }
+
+    $requiredFunctions = array('pcntl_signal', 'pcntl_signal_dispatch', 'pcntl_fork');
+    $disabled = explode(',', ini_get('disable_functions'));
```

Comment:
would `function_exists()` be sufficient if disabled in ini? Would be simpler?

Reply:
Only function_exists check is not enough. Solution was taken [here](https://stackoverflow.com/a/3938155).